src/command.c

```diff
@@ -12705,9 +12742,11 @@ else if (!strcmp(helpTkn, "Set"))
         MrBayesPrint ("      Cppmultdev      -- Standard dev. of CPP rate multipliers (log scale)       \n"); 
         MrBayesPrint ("      Cppevents       -- CPP events                                              \n"); 
         MrBayesPrint ("      TK02var         -- Variance increase in TK02 relaxed clock model           \n"); 
+        MrBayesPrint ("      WNvar           -- Variance increase in WN relaxed clock model             \n");
         MrBayesPrint ("      Igrvar          -- Variance increase in IGR relaxed clock model            \n");
```

Comment:
Correct to "Variance in IGR relaxed clock model". There is no variance increase with branch length in this model.